Nos/Service/Relay/RelayService.swift

```diff
@@ -693,6 +693,7 @@ extension RelayService {
         }
         
         for subscription in await subscriptions.active where subscription.relayAddress == client.url {
+            Log.info("Running requestEvents for subscription \(subscription)")
```

Comment:
Can you remove this log line? It's is helpful to see what is going on but we've had a problem with our logs being too noisy and when people share logs with us we only get the last few minutes worth because the system only keeps so many lines.